### PR TITLE
Bug/moowiz/api staff modification

### PR DESCRIPTION
**Pull request by moowiz**
Fixes some pylint stuff, and fixes some the `add_staff` method.

### CODE REVIEW

File: server/app/api.py [hunk 1]
```diff
@@ -1268,8 +1269,7 @@ def add_staff(self, course, user, data):
 
         if data['staff_member'] not in course.staff:
             user = models.User.get_or_insert(data['staff_member'].id())
-            course.staff.append(user.key)
-            course.put()
+            Participant.add_role(user, course, constants.STAFF_ROLE)
```
Comment (Some3141) [inline server/app/api.py]: I think we also need to add to course.staff unless we have an @property method?

**Comment by Some3141** [comment]
LGTM. There's some travis bugs.

File: server/app/api.py (hunk 1 above)
Comment (moowiz) [inline server/app/api.py]: It's a property method. It didn't used to be.

**Comment by moowiz** [comment]
Merging.

**Pull request merged**